src/app/playlist/[id]/page.tsx

```diff
@@ -92,7 +92,7 @@ const PlaylistDetail: React.FC = () => {
             >
               <FaEllipsisV size={24} color='white' />
               {showOptionsMenu && (
-                <div className='absolute top-8 right-0 bg-white/10 text-white p-2 w-40 rounded-lg shadow-lg'>
+                <div className='absolute top-8 right-0 bg-white/50 text-white p-2 w-40 rounded-lg shadow-lg'>
```

Comment:
Looks good to me. Please add a screenshot to see the changes

Reply:
```suggestion
                <div className='absolute top-8 right-0 bg-white/60 text-white p-2 w-40 rounded-lg shadow-lg'>
```

Reply:
<img width="877" alt="Screenshot 2024-12-22 at 10 05 49" src="https://github.com/user-attachments/assets/215cf0ac-688e-4ef7-b778-438b59f9f5a5" />